app/bg/dbs/settings.js

```diff
@@ -57,10 +63,12 @@ export const once = events.once.bind(events)
 
 /**
  * @param {string} key
- * @param {string | number} value
+ * @param {string | number | object} value
  * @returns {Promise<void>}
  */
 export async function set (key, value) {
+  if (['default_search_engines', 'selected_search_engine'].includes(key))
```

Comment:
Does `selected_search_engine` need to be JSONified? It's a number, right? Only objects need to be serialized.

Reply:
Right, Sqlite allows different datatypes in a column... I thought they would all have to be/would be converted to strings.